build.gradle

```diff
@@ -14,18 +17,38 @@ plugins {
 }
 
 group 'org.wycliffeassociates.translationrecorder'
-version '1.0-SNAPSHOT'
+
 
 apply plugin: 'kotlin'
+apply plugin: 'kotlin-kapt'
 
 sourceCompatibility = 1.8
 
 repositories {
     mavenCentral()
 }
+allprojects {
+    repositories {
+        maven { url "https://jitpack.io" }
+    }
+}
+allprojects {
+    repositories {
+        maven { url "https://jitpack.io" }
+    }
+}
 
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    compile group: 'org.xerial', name: 'sqlite-jdbc', version: "$sqlite_jdbc"
+    compile "io.reactivex.rxjava2:rxkotlin:$rxkotlin_version"
+    compile "io.requery:requery:$requery_version"
+    compile "io.requery:requery-kotlin:$requery_version"
+    compile 'com.google.dagger:dagger:2.16'
```

Comment:
need to add these too: 
compile "com.squareup.retrofit2:retrofit:2.0.0"
compile "com.squareup.retrofit2:converter-moshi:2.0.0"
compile 'com.squareup.retrofit2:adapter-rxjava2:2.3.0'